prepare_release.sh

```diff
@@ -123,8 +131,49 @@ update_embedded_files() {
 		Core/trackerData.json \
 		Core/AppPrivacyConfigurationDataProvider.swift \
 		Core/ios-config.json
-	eval git commit -m \"Update embedded files\" "$mute" || printf "\n✅ No changes to embedded files\n"
-	echo "✅"
+	if [[ "$(git diff --cached --exit-code)" ]]; then
+		eval git commit -m \"Update embedded files\" "$mute"
+		echo "✅"
+	else
+		printf "\nNo changes to embedded files ✅\n"
+	fi
+}
+
+update_metadata() {
+	echo "Updating metadata files ... "
+	local destination="fastlane/metadata/"
+	if [[ "${metadata}" == *.zip ]]; then
+		mkdir temp_metadata
+		unzip "${metadata}" -d "temp_metadata"
+		rsync -a --delete temp_metadata/*/ "${destination}"
```

Comment:
The `*` is problematic here. The archive you’ve given me for testing contains an additional __MACOSX folder that gets synced into fastlane directory.